src/components/LoanCollections/KivaClassicLoanCarouselExp.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+	<div class="tw-pt-4">
+		<transition name="kvfade">
+			<div
+				v-if="isLoading"
+				class="spinner"
+			>
+				<kv-loading-spinner />
+			</div>
+		</transition>
+
+		<kv-carousel
+			v-if="augmentedLoanIds.length > 0 && isVisible"
+			class="tw-w-full tw-overflow-visible md:tw-overflow-hidden"
+			:embla-options="{
+				loop: false,
+			}"
+			ref="categoryCarousel"
+			:multiple-slides-visible="true"
+			slides-to-scroll="visible"
+			:slide-max-width="singleSlideWidth"
+			@interact-carousel="onInteractCarousel"
+		>
+			<template v-for="(loanId, index) in augmentedLoanIds" #[`slide${index}`]>
+				<!-- Show View more Card -->
+				<router-link
+					v-if="loanId === 3"
+					:key="`view-more-card-${loanId}`"
+					class="tw-flex tw-items-center tw-h-full tw-w-full
+						hover:tw-bg-action-highlight hover:tw-text-primary-inverse tw-rounded"
+					:to="cleanUrl"
+					v-kv-track-event="[
+						'Homepage',
```

Comment:
We can update tracking params as part of the tracking specific ticket if you'd like. The category would become `Monthly Good`